tutorials/evoked/30_eeg_erp.py

```diff
@@ -373,7 +374,204 @@
 #
 # The tutorials :ref:`tut-epochs-class` and :ref:`tut-evoked-class` have many
 # more details about working with the `~mne.Epochs` and `~mne.Evoked` classes.
+
+# %%
+# Amplitude and Latency Measures
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# It is common in ERP research to extract measures of amplitude or latency to
+# compare across different conditions. There are many measures that can be
+# extracted from ERPs, and many of these are detailed (including the respective
+# strengths and weaknesses) in Ch. 9 of Luck :footcite:`Luck2014` (also see
+# the `Measurement Tool <https://bit.ly/37uydRw>`_ in the ERPLAB Toolbox
+# :footcite:`Lopez-CalderonLuck2014`).
+#
+# This part of the tutorial will demonstrate how to extract three common
+# measures:
+#
+# * Peak Latency
+# * Peak Amplitude
+# * Mean Amplitude
+#
+# Peak Latency and Amplitude
+# --------------------------
+#
+# Probably most common measures of amplitude and latency are peak measures.
+# Peak measures are basically the maximum amplitude of the signal in a
+# specified time window, and the time point (or latency) at which the peak
+# amplitude occurred.
+#
+# Peak measures can be obtained using the `~mne.Evoked.get_peak` method. There
+# are two important things to point out about `~mne.Evoked.get_peak` method.
+# First, it returns the peak latency and amplitude from **all channels** in
+# the `~mne.Evoked` object.
```

Comment:
It is not clear to me if just one peak latency and amplitude is returned, or one for each channel.

Reply:
I will update this to be more clear. At present, it returns only 1 peak and latency, which is the highest amplitude peak across all channels.

Reply:
```suggestion
# First, it finds the strongest peak looking across **all channels** of
# the selected type that are available in the `~mne.Evoked` object. As a
# consequence if you want to restrict the search for the peak to a group of
# channels, you should first use `~mne.Evoked.pick_channels`.
```